vhdbuilder/packer/configure-windows-vhd.ps1

```diff
@@ -119,9 +119,13 @@ function Get-FilesToCacheOnVHD {
 
         foreach ($URL in $map[$dir]) {
             $fileName = [IO.Path]::GetFileName($URL)
-
+            # Doesnot cache containerd package in docker image
+            if ($containerRuntime -ne 'containerd' -And $dir -eq "c:\akse-cache\containerd\") {
+                Write-Log "Skip to download $URL for docker image"
+                continue
+            }
```

Comment:
may less confusing
```suggestion
            # Does not cache containerd packages in docker VHD
            if ($containerRuntime -eq 'containerd' -And $dir -eq "c:\akse-cache\containerd\") {
                Write-Log "Skip to download $URL for docker VHD"
                continue
            }
```

Reply:
Cannot understand. Can you explain a bit more?
Why do we need to skip `"c:\akse-cache\containerd\"` when $containerRuntime -eq 'containerd'?

Reply:
Updated the comments per your suggestion. But I think that we should not change `ne` to `eq`